React/Base/RCTConvert.m

```diff
@@ -567,6 +571,25 @@ static BOOL RCTFontIsCondensed(UIFont *font)
   return (symbolicTraits & UIFontDescriptorTraitCondensed) != 0;
 }
 
+NSDictionary *RCTFontVariantMapping = NULL;
+
++ (NSDictionary *)getRCTFontVariantMapping
```

Comment:
just make this a static function and use dispatch_once to initialise it just once. In Objective-C we also only use get as a prefix when the result will be returned indirectly.